src/main/java/com/zufar/icedlatte/security/api/UserAuthenticationService.java

```diff
@@ -24,11 +25,13 @@ public class UserAuthenticationService {
     private final AuthenticationManager authenticationManager;
     private final LoginFailureHandler loginFailureHandler;
     private final ResetLoginAttemptsService resetLoginAttemptsService;
+    private final TokenManager tokenManager;
+
+    int userAccountLockoutDurationMinutes = 30;
```

Comment:
nit: since this is a constant, I would rename to `private static final Integer ACCOUNT_LOCKOUT_DURATION_MINUTES`

Reply:
Fixed